central/complianceoperator/v2/compliancemanager/manager_impl_test.go

```diff
@@ -423,7 +423,10 @@ func (suite *complianceManagerTestSuite) TestUpdateScanRequest() {
 			setMocks: func() {
 			},
 			isErrorTest: true,
-			expectedErr: fmt.Sprintf("Scan Configuration ID is required for an update, scan_config_name:%q profiles:<profile_name:\"ocp4-cis\" > ", mockScanName),
+			expectedErr: fmt.Sprintf("Scan Configuration ID is required for an update, %+v", &storage.ComplianceOperatorScanConfigurationV2{
```

Comment:
nit (code duplication): can the ComplianceOperatorScanConfigurationV2 be extracted to a file variable ?